azavea/district-builder-dtl-pa
# Change competitiveness metric

## Overview

Change competitiveness metric for 17 district legislative body to conform to C70's new definition (`|D% - R%| <= 10`).

This updates competitiveness in both the side panel and the leaderboard.

### Checklist


## Testing Instructions

* Check out `develop` and run `./scripts/configure_pa_data` to start from a clean slate
* Run `./scripts/update` and then `./scripts/server`
* View the '17 Districts Congressional' map and note the number of competitive districts (it currently has 5 competitive districts, but the default template will soon change as per https://github.com/azavea/district-builder-dtl-pa/pull/148#event-2128298929)
* Check out this branch
* Run `docker-compose exec django bash` and then `./manage.py removescoreconfig` and `./manage.py setup config/config.xml -f` within the container (this will need to be run manually in staging and production for the config changes to take effect)
* View the same plan and it should have 7 competitive districts as per the changes to how competitiveness is defined
* Submit the plan and make sure it has 7 competitive districts on the leaderboard as well 

Closes #163527407
